custom_components/tuya_local/devices/sime_ecomaxi_vb200_waterheater.yaml

```diff
@@ -0,0 +1,181 @@
+name: Heat pump water heater
+products:
+  - id: ukqqxsk7vfz8co9z
+    manufacturer: SIME
+    model: ECOMAXI VB 200 WiFi
+entities:
+  - entity: water_heater
+    dps:
+      - id: 1
+        name: operation_mode
+        type: boolean
+        mapping:
+          - dps_val: false
+            value: "off"
+            icon: "mdi:water-boiler-off"
+          - dps_val: true
+            value: heat_pump
+            icon: "mdi:water-boiler"
+      - id: 4
+        name: temperature
+        type: integer
+        unit: C
+        range:
+          min: 35
+          max: 65
+      - id: 16
+        name: current_temperature
+        type: integer
+        unit: C
+  - entity: select
+    name: Operating mode
+    category: config
+    dps:
+      - id: 2
+        name: option
+        type: string
+        mapping:
+          - dps_val: "0x00"
+            value: auto
+            icon: "mdi:autorenew"
+          - dps_val: "0x02"
+            value: "heat pump"
```

Comment:
Since this no longer needs to correspond to the HA water_heater supported operation_modes, you could go back to the original names if that is more obvious - in line with manual and Tuya app.